nutanix/data_source_nutanix_ndb_databases.go

```diff
@@ -229,3 +247,14 @@ func flattenDatabaseIntancesList(db *era.ListDatabaseInstance) []map[string]inte
 	}
 	return nil
 }
+
+func filterDatabaseBasedOnDatabaseEngine(resp *era.ListDatabaseInstance, dbengine string) (*era.ListDatabaseInstance, error) {
+	found := make(era.ListDatabaseInstance, 0)
+
+	for _, v := range *resp {
+		if dbengine == v.Type {
```

Comment:
@abhimutant Please convey this to ERA team to support this as a parameter.

Reply:
It's already conveyed and we are tracking this.